### ISSUE TITLE
css.walkRules is not a function

### ISSUE THREAD

**Issue by enguerran** [cssnext/cssnext#200]
I got this error message: 

```
% cssnext style.css output.css

cssnext encounters an error:

css.walkRules is not a function
undefined

If this error looks like a bug, please report it here:
❯ https://github.com/cssnext/cssnext/issues
```

Steps to reproduce are easy:

```
mkdir cssnext-issue && cd cssnext-issue
touch style.css
```

style.css contains the following:

```css
:root {
  --vr-Color-red: #f00;
}

h1 {
  color: var(--vr-Color-red);
}
```

Then just install and run the CLI:

```
npm i -D cssnext
cssnext style.css output.css
```

Result:
![screenshot from 2015-08-24 16 39 43](https://cloud.githubusercontent.com/assets/701648/9442974/d2dd46f4-4a7e-11e5-98b2-ea7e4e4b13bb.png)

Versons:
```
cssnext --version
1.8.3
node --version
v0.12.7
npm --version
2.11.3
```

Maybe a dependency is causing this issue, is not it?

**[EDIT]**
It seems that the error message can be either:

* `css.walkRules is not a function`
* `undefined is not a function`

**Comment by MoOx** [comment]
It looks like a dep is now using postcss v5.x but has introduced this upgrade has a minor release...

It seems that someone didnt respect semver...

I am on vacations so I cannot handle this atm. PR welcome.

**Comment by bloodyowl** [comment]
cannot reproduce here

<img width="762" alt="screen shot 2015-08-24 at 16 52 27" src="https://cloud.githubusercontent.com/assets/1688645/9443278/83d8ddbe-4a80-11e5-81eb-b3a9b868b82a.png">

**Comment by magsout** [comment]
WFM 

![capture d ecran 2015-08-24 a 16 56 38](https://cloud.githubusercontent.com/assets/1997108/9443395/2120799c-4a81-11e5-94d9-4e4250c85b32.png)

**Comment by enguerran** [comment]
May it be https://github.com/robwierzbowski/node-pixrem?
:'(

![screenshot from 2015-08-24 17 12 00](https://cloud.githubusercontent.com/assets/701648/9443779/5b0bb4b2-4a83-11e5-9821-7d61b0a8ef74.png)

**Comment by enguerran** [comment]
I try to follow https://github.com/cssnext/cssnext/blob/master/CONTRIBUTING.md but I cannot `npm install` as I get this following error:

```
npm ERR! peerinvalid The package eslint does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-loader@0.14.2 wants eslint@0.21 - 0.24 || ~1.0.0-rc-0
```

Any clue?

**Comment by enguerran** [comment]
Ok, So I `npm install`ed the project by modifying `"eslint": "^1.0.0-rc-1"` to `"eslint": "~1.0.0-rc-1"`. But now I get the following error with `npm test`:

```
config.__SERVER_URL__ = `${config.__SERVER_PROTOCOL__}${config.__SERVER_HOST__
                        ^
SyntaxError: Unexpected token ILLEGAL
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/enguerran/Development/cssnext/webpack.config.js:4:21)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
...
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --output-library-target umd --output-library cssnext dist/index.js dist/cssnext.js
```

Any clue?

**Comment by enguerran** [comment]
Ok, so I fixed this by using `io/2.3.0` instead of `node/0.12.7`. PR is coming...

**Comment by magsout** [comment]
See https://github.com/cssnext/cssnext/pull/201#issuecomment-134341131
